linode_api4/objects/serializable.py

```diff
@@ -223,3 +224,22 @@ def __delitem__(self, key):
 
     def __len__(self):
         return len(vars(self))
+
+
+class StrEnum(str, Enum):
+    """
+    Used for enums that are of type string, which is necessary
+    for implicit JSON serialization.
+
+    NOTE: Replace this with StrEnum once Python 3.10 has been EOL'd.
+    See: https://docs.python.org/3/library/enum.html#enum.StrEnum
```

Comment:
Lots of feature are not in older Python versions, and there are still 2 years before 3.10 retired. Let's just wait...